.github/workflows/continuous-integration.yml

```diff
@@ -1,81 +1,78 @@
 name: Continuous Integration
 
 on:
-    push:
-        branches:
-            - master
-            - 6.0
-            - 5.0
-            - 4.0
-            - 3.0
-            - 2.5
-    pull_request:
-        types: [ opened, synchronize, reopened ]
+  push:
+    branches:
+      - master
```

Comment:
since we will open PRs upon version branches, we should keep them in this list.

Reply:
I'm not really sure if that will work. The master branch has a higher PHP version that the other branches

Reply:
not sure either... let's see...

Reply:
We will see when we actually open a PR on one of those branches. Then we will check what can we do to solve it